.github/workflows/push_mirror.yml

```
@@ -0,0 +1,15 @@
+name: Mirroring
+
+on: [push, delete]
+
+jobs:
+  to_gitlab:
+    runs-on: ubuntu-18.04
+    steps:                                              # <-- must use actions/checkout@v1 before mirroring!
+    - uses: actions/checkout@v1
+    - uses: pixta-dev/repository-mirroring-action@v1
+      with:
+        target_repo_url:
+          git@gitlab.pnnl.gov:2222/exasgd/solvers/hiop.git
```

```suggestion
          git@gitlab.pnnl.gov:2222/exasgd/solvers/hiop-mirror.git
```
This should go to HiOp mirror repository at our GitLab.

---

Otherwise looks good.

---

Oh.. ok interesting I thought we might just want to push this to the existing code repo (non `mirror` variant). So you want to maintain that `-mirror` variant. Understood. I'll make the change.

---

Done, ready for review @pelesh